python/lsst/rubintv/analysis/service/commands/db.py

```diff
@@ -212,7 +211,53 @@ def build_contents(self, data_center: DataCenter) -> dict:
         return result
 
 
+@dataclass(kw_only=True)
+class AggregateQueryCommand(BaseCommand):
+    """Perform the given aggregate query on database tables.
+
+    Attributes
+    ----------
+
+
+    """
+
+    database: str
+    columns: list[str]
+    query_type: str
+    query: dict | None = None
+    global_query: dict | None = None
+    day_obs: str | None = None
+    data_ids: list[tuple[int, int]] | None = None
+    response_type: str = "aggregate"
+
+    def build_contents(self, data_center: DataCenter) -> dict:
+        """Query the database to count rows in each specified table."""
+        database = data_center.schemas[self.database]
+
+        # Initialize a dictionary to store row counts
+        result = {}
+
+        for column in self.columns:
+            # Construct an AggregateQuery for counting non-NULL rows
+            aggregate_query = AggregateQuery(
+                aggregate=self.query_type,
+                column=column,
+            )
+
+            # Execute the AggregateQuery directly to get the count
+            query_result = aggregate_query(database)
+
+            # Extract the row count from the query result
+            result[column] = query_result.result[self.query_type]
+
+        return {
+            "schema": self.database,
+            self.response_type: result,
```

Comment:
So the response type is added to the message that is built by the `execute` command (see https://github.com/lsst-ts/rubintv_analysis_service/blob/main/python/lsst/rubintv/analysis/service/command.py#L154-L166). So I think that the return should be
```
{
    "schema": self.database,
    self.query_type: result,
}
```
that way rubintv_visualization will know which function was used.

Reply:
Yes, of course. I was imagining that the command that sent the message to the workers would keep its own track of returns and that we'd have a `CountRowsCommand`, `MaxRowCommand` etc. in r_v.